web/styles/modal.css

```diff
@@ -60,12 +60,18 @@
     padding-right: 1rem;
     max-width: 80%;
     display: flex;
+    align-items: center;
 
     .user_profile_name {
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
     }
+
+    #user_profile_edit_button_icon {
+        color: hsl(200deg 79% 66%);
+        cursor: pointer;
+    }
 }
```

Comment:
Should these changes be in the later commit?

Is the `cursor: pointer` here correct?

Reply:
Changed the commit and `cursor: pointer` is required so that on hover the user should feel the pencil is clickable, do let me know if you want me to change it.